flyway-community-db-support/flyway-database-yugabytedb/src/main/java/org/flywaydb/community/database/postgresql/yugabytedb/YugabyteDBDatabase.java

```diff
@@ -30,6 +36,13 @@ public YugabyteDBDatabase(Configuration configuration, JdbcConnectionFactory jdb
 
     @Override
     protected YugabyteDBConnection doGetConnection(Connection connection) {
+        Statement stmt = null;
+        try {
+            stmt = connection.createStatement();
+            stmt.execute("set yb_silence_advisory_locks_not_supported_error=on;");
+        } catch (SQLException throwables) {
+            LOG.warn("Unable to set yb_silence_advisory_locks_not_supported_error ", throwables.printStackTrace(););
```

Comment:
How does this even compile? I see two `;`.
And `printStackTrace()` would print throwable and it stack trace to the **standard error stream** which is not what we want.
```
LOG.warn("Unable to set yb_silence_advisory_locks_not_supported_error ", throwable);
```